### PR TITLE
[tr] Add Turkish localization for style-guide/_index.md

### PR DESCRIPTION
**Pull request by rwxdash**
### Describe your changes
- Added Turkish localization for `style-guide/_index.md`

Google Docs for reference
- [`style-guide/_index.md`](https://docs.google.com/document/d/1yKXJ5V5zV5NzL_79aJoynQ66vRCX7MgBg7nvSQRd6QQ/edit?usp=sharing)

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

### CODE REVIEW

**Review by halil-bugol — APPROVED** [review]

File: content/tr/style-guide/_index.md [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+
+## Tanım şablonu
+
+Her sözlük terimi bir markdown dosyasında tutulur ve aşağıdaki şablonu takip eder:
+
+```md
+---
+title: 
+status: 
+category:
```
Comment (aliok) [inline content/tr/style-guide/_index.md:48]:
```suggestion
title:
status:
category:
```

Just noticed `NBSP` chars here on my IDE

**Review by aliok — COMMENTED** [review]
Just noticed a minor issue with `NBSP` chars but LGTM otherwise.

Thanks a lot, this was a hard page to translate!

File: content/tr/style-guide/_index.md [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+Başlamadan önce, ayrıntı düzeyi, zorluk düzeyi ve örneklerin ne zaman uygun olduğu konusunda bir fikir edinmek için lütfen yayınlanmış bazı Sözlük terimlerini okuyun.
+
+## İnceleme süreci: Ne beklemeli?
+
+Lütfen boş zamanlarında bu işi yapan kısıtlı sayıda geliştiriciler olduğumuzu unutmayın.
+Zaman zaman kavramları hızlı bir şekilde gözden geçirebiliriz; diğer durumlarda ise bu biraz zaman alabilir.
+Sabrınız için teşekkür ederiz.
+Herhangi bir sorunuz olursa, lütfen [#glossary](https://cloud-native.slack.com/archives/C02TX20MQBB) Slack kanalından,
+Türkçe çeviriler için ise [#glossary-localizations-turkish](https://cloud-native.slack.com/archives/C05ATBGLHJ7) kanalından,
+bizimle iletişime geçin (nerede ve nasıl bulacağınız için lütfen [Nasıl Katkıda Bulunulur](/tr/contribute/) belgemize bakın).
```
Comment (aliok) [inline content/tr/style-guide/_index.md:180]: Great stuff!

File: content/tr/style-guide/_index.md (hunk 1 above)
Comment (rwxdash) [reply content/tr/style-guide/_index.md:48]: Must have come from the gdoc2md. Thank you for catching this.

**Review by aliok — APPROVED** [review]
LGTM, thanks, great work!

**Pull request merged**